views/extras.tmpl

```diff
@@ -194,36 +194,26 @@
         console.log('Starting websocket');
         console.log("createdWebSocket", uri, ws)
         ws.registerEvtHandler("open", function() {
```

Comment:
Just for clarification, this console logs are used especially in debugging. That said some of the logging available now are fairly useless and some are too important to be deleted. 
We could do away with quite a number of them but logging for New Blocks and New Transactions notifications should be maintained.